examples/rust/api_demo/src/main.rs

```diff
@@ -251,7 +251,7 @@ fn demo_rects(rec_stream: &RecordingStream) -> anyhow::Result<()> {
     use ndarray_rand::{rand_distr::Uniform, RandomExt as _};
 
     // Add an image
-    let img = Array::<u8, _>::from_elem((1024, 1024, 3).f(), 128);
+    let img = Array::<u8, _>::from_elem((1024, 1024, 3, 1).f(), 128);
```

Comment:
This doesn't seem like it belongs in the example?

Reply:
I needed somewhere to actually see a tensor like that in action and I didn't find anywhere else appropriate in our samples. If you mind that too much I can remove it again, but would be nice to have a trailing dimension somewhere in the samples 🤔